src/Backend/Modules/FormBuilder/Actions/Edit.php

```diff
@@ -174,6 +174,21 @@ function ($value) {
         $this->form->addText('datetime_classname');
         $this->form->addText('datetime_error_message');
 
+        // map the values to replace them with the backend translations
+        // use array combine to set the keys as the autocomplete values instead of key values
+        $this->form->addDropdown(
+            'datetime_autocomplete',
+            array_map(
```

Comment:
Since you do this map twice (once in Add, once in Edit), it might be a good idea to extract it to a method in Autocomplete, `Autocomplete::getValuesForDropdown()` for example.

Reply:
The edit action is meant for editing a formbuilder form and not the field.
That means the dropdown is only added in the edit action.

Reply:
My bad! Either way, you are doing the same processing of values twice so you should extract it to a single method in the `Autocomplete` class.